dev/devicelab/bin/tasks/run_release_test.dart

```diff
@@ -114,8 +114,8 @@ void main() {
 
       _findNextMatcherInList(
         stdout,
-        (String line) => line.startsWith('Installing build/app/outputs/flutter-apk/app.apk...'),
-        'Installing build/app/outputs/flutter-apk/app.apk...',
+        (String line) => line.startsWith('Installing build/app/outputs/flutter-apk/app-release.apk...'),
```

Comment:
For posterity, this is the new fix for https://github.com/flutter/flutter/issues/113620 that caused the previous PR to get reverted.